drivers/storage/portworx/preflight.go

```diff
@@ -369,6 +380,13 @@ func (u *preFlightPortworx) processNodesChecks(recorder record.EventRecorder, st
 		}
 	}
 
+	if isFatal { // Add events for fatal checks
+		for _, fmsg := range fmsgs {
+			k8sutil.WarningEvent(recorder, u.cluster, util.FailedPreFlight, fmsg)
+		}
```

Comment:
The loop will process checks looking for any failures. However with this PR there is a difference between a failure and "fatal" failure. So I wanted to make sure the fatal events are printed after all the other failed events. So I collected the fatal msgs so they can be sent all at once instead of being intermixed with other failed events.